crates/messages/proto/messages.proto

```diff
@@ -31,11 +19,30 @@ message WorkerState {
 }
 
 message DatasetRanges {
+  option deprecated = true;
+
   string url = 1;
   repeated Range ranges = 2;
 }
 
+message BitString {
+  bytes data = 1;
+  uint64 size = 2;
+}
+
+// Worker -> Scheduler, Portal, Ping collector
 message Ping {
```

Comment:
@denisbsu suggested renaming it because it's not really a ping anymore (it doesn't get an answer).\
Possible candidates are "Heartbeat", "WorkerState", "Availability", "IHave"\
I personally like "Heartbeat"

Reply:
>I personally like "Heartbeat"

Yes, I like this one as well